web/src/components/questions/Questions.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) [2022-2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import GenericQuestion from "~/components/questions/GenericQuestion";
+import QuestionWithPassword from "~/components/questions/QuestionWithPassword";
+import LuksActivationQuestion from "~/components/questions/LuksActivationQuestion";
```

Comment:
We've agreed on not exporting all components in index file ( see https://trello.com/c/c0GEUMQ8 (internal link) and https://github.com/openSUSE/agama/issues/1049). And this is a little step towards such an agreement. However, looking at it, two questions comes to my mind:

* Should we use `~/components/<namespace>/<Component>` or `./<Component>`. I see the first approach kind of _safe_ and _stable_, since it does not matter from where you are importing the component.
* If we use ``~/components/<namespace>/<Component>` does make sense to keep exporting _public_ components in the index file. For example, for importing `Questions.tsx` we could

  ```js
  import Questions from "~/components/questions/Questions";
  ```
  or 

  ```js
   import { Questions } from "~/components/questions";

Honestly, for namespaces exporting a single component there is not too much difference / advantages. Another story is when importing multiple components from a single namespace, like _core_. Similar to what happens when importing components from PatternFly: it would be really annoying if we have had to import each component from its path.

Just sharing thoughts. No strong opinion here (yet :stuck_out_tongue_closed_eyes:)
  ```